docs/source/developing/ci.adoc

```diff
@@ -10,21 +10,51 @@ toc::[]
 [[ci-overview]]
 == Overview
 
-The following sections describes the various CI setups Minishift utilizes.
+Minishift uses several CI build servers such as Centos, AppVeyor, Circle and Travis. These servers build incoming pull requests, pass it through various test parameters and provide the status of the builds to ensure continuous integration and deployment. Except for Travis CI, they also archive build artifacts of successful builds.
 
-[[ci-setup]]
+The following sections describe the various CI build servers Minishift utilizes.
+
+[[centos-ci]]
 == CentOS CI
 
-Minishift uses several CI build servers, including https://ci.centos.org/[CentOS CI]. The server
-builds incoming pull requests and pushes to master. It also archives build artifacts of successful
-builds.
+The https://ci.centos.org/[CentOS CI] server is used specifically to test integration tests, unit tests, code format tests and cross build tests in Linux (CentOS).
 
 You can find the CentOS CI Jenkins job for Minishift https://ci.centos.org/job/minishift/[here].
 
 The artifacts of a successful pull request build can be found at
 http://artifacts.ci.centos.org/minishift/minishift/pr/[artifacts.ci.centos.org/minishift/minishift/pr/<PRID>].
 
-Artifacts of a successful master build can be found at
+The artifacts of a successful master build can be found at
 http://artifacts.ci.centos.org/minishift/minishift/master/[artifacts.ci.centos.org/minishift/minishift/master/<BUILDID>].
 
 For more information about CentOS CI, see the https://wiki.centos.org/QaWiki/CI[CentOS CI] documentation.
+
+[[circle-ci]]
+== Circle CI
+
+The https://circleci.com/[Circle CI] is used for testing unit tests, code format tests and cross build tests in Mac OS
+
+You can find the circle-CI job for Minishift  https://circleci.com/gh/minishift/minishift[here].
+
+The artifacts of a successful pull request build can be found at https://circleci.com/gh/minishift/minishift/<build_id>#artifacts/ in the _distiller/src/github.com/minishift/minishift/out/_ directory.
+
+Screenshot
```

Comment:
> But, based on your points above this does not seem that important considering we are advertising CentOS CI. So maybe we can drop this.

+1, I don't think it adds something here. We should use CentOS CI for all of this. 

> This one could fit in after line 13, giving a context of what we are referring to. However if you think most developers interested in developing Minishift would find this elementary, we could drop this too.

This one might be nice. I don't think it is super important, but it visually supports what we are talking about. +1